README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
@atarzwell You're right, it's rare for personal or political matters to ever be brought up in issue pages.  Though, it is beginning to happen more due to the proliferation of [novel projects outside the realm of strictly coding.](http://www.wired.com/2013/03/series-seed/)  Right now it appears Github has a low threshold of tolerance before it will act on removing a pages it finds questionable. [This controversial satirical coding language being a prominent recent example.](http://knowyourmeme.com/memes/events/c-plus-equality-c)

-----------
Anyways, as for the actual line of text in question, I question the need for this FAQ question section to even exist.   The "question" as it is originally written is really just a set up to make a statement:

```
#### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 		 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
```

I recommend replacing the whole `free speech` question with a more general statement elsewhere about how the Code of Conduct strives to uphold a dual commitment to freedom of expression and the protection of people from targeted harassment.  Even a general statement like this could work:

```
We value freedom of expression on the internet; but with rights comes responsibilities. In short: [Don't be an asshole](https://en.wikipedia.org/wiki/The_No_Asshole_Rule). Act in good faith.  Follow the [golden rule](https://en.wikipedia.org/wiki/Golden_Rule).
```